src/translations/main_it.ts

```diff
@@ -0,0 +1,905 @@
+    </message>
+</context>
+<context>
+    <name>QObject</name>
+    <message>
+        <source>Account</source>
+        <translation>Account</translation>
+    </message>
+    <message>
+        <source>Notification color</source>
+        <translation>Colore di notifica</translation>
+    </message>
+</context>
+<context>
+    <name>TrayIcon</name>
+    <message>
+        <source>Would you like to set up Birdtray?</source>
+        <translation>Vuoi installare Birdtray?</translation>
+    </message>
+    <message>
+        <source>You have not yet configured any email folders to monitor. Would you like to do it now?</source>
+        <translation>Non hai ancora configurato alcuna cartella di posta elettronica da monitorare. Ti piacerebbe farlo ora?</translation>
+    </message>
+    <message>
+        <source>Show Thunderbird</source>
+        <translation>Mostra Thunderbird</translation>
+    </message>
+    <message>
+        <source>Hide Thunderbird</source>
+        <translation>Nascondi Thunderbird</translation>
+    </message>
+    <message>
+        <source>New Email</source>
+        <translation>Nuova Email</translation>
+    </message>
+    <message>
+        <source>Blank</source>
+        <translation>Vuota</translation>
+    </message>
+    <message>
+        <source>New Email Message</source>
+        <translation>Nuovo messaggio e-mail</translation>
+    </message>
+    <message>
+        <source>5 minutes</source>
+        <translation>5 minuti</translation>
+    </message>
+    <message>
+        <source>10 minutes</source>
+        <translation>10 minuti</translation>
+    </message>
+    <message>
+        <source>30 minutes</source>
+        <translation>30 minuti</translation>
+    </message>
+    <message>
+        <source>1 hour</source>
+        <translation>1 ora</translation>
+    </message>
+    <message>
+        <source>4 hours</source>
+        <translation>4 ore</translation>
+    </message>
+    <message>
+        <source>Snooze for ...</source>
+        <translation>Posticipa per...</translation>
+    </message>
+    <message>
+        <source>Unsnooze</source>
+        <translation>Non posticipare</translation>
+    </message>
+    <message>
+        <source>Ignore unread emails</source>
+        <translation>Ignora le email non lette</translation>
+    </message>
+    <message>
+        <source>Settings...</source>
+        <translation>Impostazioni...</translation>
+    </message>
+    <message>
+        <source>Quit</source>
+        <translation>Esci</translation>
+    </message>
+    <message>
+        <source>Cannot start Thunderbird</source>
+        <translation>Impossibile avviare Thunderbird</translation>
+    </message>
+    <message>
+        <source>Error starting Thunderbird, because we could not attach to the updater:
+
+%1</source>
+        <translation>Errore durante l'avvio di Thunderbird, perché non è stato possibile collegarsi al programma di aggiornamento:
+
+ %1</translation>
```

Comment:
```suggestion
%1</translation>
```
There is an additional space at the start that will offset the text behind it.

Reply:
fixed!
I don't see the missing translation.
Let me know where for fix it.
Tx.